### ISSUE TITLE
Checking metadata on key check page

### ISSUE THREAD

**Issue by KevinThiele** [AtlasOfLivingAustralia/ala-keys-ui#19]
It would be good to display the citation for the key at the top of the key check page, to allow a visual check of the metadata as well as the key itself.

**Comment by nielsklazenga** [comment]
Key check page?

**Comment by KevinThiele** [comment]
The page that comes after you've hit Submit on the New Key page

**Comment by nielsklazenga** [comment]
That page only shows when you upload a new key file, not when you just change the metadata. When the key file structure is right, but the metadata is wrong, you won't want to go back to the Edit page, as it is easier to just upload the key and fix the metadata afterwards.

**Comment by KevinThiele** [comment]
Yes, and that's the circumstance when I think it would be useful to be able to do this. It seems to me more intuitive to be able to check the metadata at key upload time, and if necessary go back to correct any errors, rather than go ahead and upload the key, then go to edit to correct any metadata errors. It's a relatively small thing and not something I'll lose sleep over, but it makes sense to me. My thought would be simply to have the same line we currently have when the key is displayed (e.g. "Modified from: Kubitzki. K. & Bayer, C. (2003). Malvaceae. In: Kubitzki, K. (ed), The Families and Genera of Vascular Plants V. Springer, Berlin.") at the head of that key check page. 

But maybe this would not be straightforward, as you don't yet have the metadata in the database at this time, so it would be hard to construct the string? If that's the case, then no problem, ignore this suggestion.

**Comment by nielsklazenga** [comment]
It doesn't make sense in the flow of the application. Editing/adding the key and editing/adding the metadata are two entirely different things that shouldn't be confuddled.

**Comment by KevinThiele** [comment]
But to me it does make sense in the flow of the way the application's used. When I add a new key, the current flow is:

1. Enter the key's metadata
2. Attach the key
3. Check the key
4. Everything OK? Go ahead

I'm just suggesting adding the ability to do another check:

1. Enter the key's metadata
2. Attach the key
3. Check the key and its metadata
4. Everything OK? Go ahead

**Comment by nielsklazenga** [comment]
That is almost what I have in mind. Hopefully this week.
